translations/case+intl-icu.da.xlf

```diff
@@ -1265,6 +1265,10 @@
         <source>Create party</source>
         <target state="translated">Opret part</target>
       </trans-unit>
+      <trans-unit id="EE6DD9s" resname="Extra complaint category information">
+        <source>Extra complaint category information</source>
+        <target state="translated">Ekstra klagetype information</target>
```

Comment:
“klagetypeinformation” er ét ord.

Reply:
Fixed in 84aa41c02c382aef84288c651f1429f4922bd6d3.